fixtures/services/hazardous-waste-dropoff.yaml

```diff
@@ -54,6 +55,14 @@ additional_content_ar: |
     <p>لا تخلط العناصر. لن يتم قبول الأحمال الكبيرة للشاحنات أو المقطورة بعد الساعة 4:45 مساءً.</p>
     <p>الشركات ليست مؤهلة للتخلص من النفايات الخطرة. للحصول على معلومات حول كيفية تخلص الشركات من النفايات الخطرة، اتصل على الرقم 512-974-4336.</p>
 
+dynamic_content:
+    -
+        type: what_do_i_do_with
```

Comment:
same as above -- We'll need to update the switch statement in src/js/modules/ApplicationBlock

Reply:
This won't change the API response